BookingsApi/BookingsApi.DAL/Migrations/BookingsDbContextModelSnapshot.cs

```diff
@@ -696,6 +705,9 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<int?>("InterpreterLanguageId")
                         .HasColumnType("int");
 
+                    b.Property<string>("MeasuresExternalId")
```

Comment:
Should we start to have a limited nvarchar for performance purpose? I know all the other fields are to max but maybe something we should start looking into it.